funnel/models/commentvote.py

```diff
@@ -96,7 +100,7 @@ class Vote(NoIdMixin, db.Model):
     votedown = db.Column(db.Boolean, default=False, nullable=False)
 
     @classmethod
-    def migrate_user(cls, old_user, new_user):
+    def migrate_user(cls, old_user: User, new_user: User) -> Optional[Iterable[str]]:
```

Comment:
`migrate_user` optionally returns a list of tables that have been migrated. This return type here is confusing. It should be defined in a single place (a types module?) as `OptionalMigratedTables`.